src/components/ArtistName.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+export const ArtistName = ({ artists }) => {
+  return (
+    <span>
+      {artists.map((artist, index) => (
+        <span key={index}>
+          <a key={artist.id} href={artist.external_urls.spotify}>
+            <span className="artist-name">
+              {artist.name} {index === artists.length - 2 && " & "}
```

Comment:
Can you think of a way to make the & separate from the links?

![Skärmavbild 2024-03-13 kl  10 55 19](https://github.com/Technigo/project-music-releases-vite/assets/51529201/641c8c19-5c01-449a-ba70-695789373dcb)